src/MPS.jl

```diff
@@ -556,4 +557,4 @@ function LinearAlgebra.normalize!(config::MixedCanonical, ψ::AbstractMPO; at=co
     return ψ
 end
 
-# TODO function LinearAlgebra.normalize!(::Canonical, ψ::AbstractMPO) end
+LinearAlgebra.normalize!(::Canonical, ψ::AbstractMPO) = canonize!(ψ; normalize=true)
```

Comment:
mmm why do you need to canonize on normalization?

Reply:
It is a practical way to do that, just that